src/lib/stellar/sep10.js

```diff
@@ -1,4 +1,4 @@
-import { Utils } from 'stellar-sdk'
+import { Utils } from '@stellar/stellar-sdk'
```

Comment:
There's one more change that I didn't catch before. We need to import `WebAuth` here, instead of `Utils`. Then, down in the `validateChallengeTransaction` function, we'll need to change `Utils.readChallengeTx` to `WebAuth.readChallenegeTx`. I believe the function signature remains the same, but it just moved from one module/class to another.

Here's the documentation on that function now: https://stellar.github.io/js-stellar-sdk/module-WebAuth.html#~readChallengeTx